packages/jest-expo/config/getPlatformPreset.js

```diff
@@ -84,6 +84,8 @@ function getPlatformPreset(displayOptions, extensions, platform, { isServer, isR
       defaultPlatform: extensions[0],
       platforms: extensions,
     },
+    // See: https://jestjs.io/docs/configuration#prettierpath-string
+    prettierPath: require.resolve('jest-snapshot-prettier'),
```

Comment:
This shouldn't be in the "public" `jest-expo` preset, since this package doesn't exist and we install it for our repo via an alias in `expo-module-scripts`. If we want to add it just for our repo, it should be only in `expo-module-scripts`, but here it should be removed (or try-catch'd, but I'm assuming this is an accidental addition?)